apps/nextjs-website/src/components/molecules/DownloadableDocuments/DownloadableDocuments.tsx

```diff
@@ -0,0 +1,85 @@
+'use client';
+import React from 'react';
+import { Box, Grid, Typography } from '@mui/material';
+import Download from '@mui/icons-material/Download';
+import CtaCard from '@/components/atoms/CtaCard/CtaCard';
+import { useTranslations } from 'next-intl';
+
+const getReadableFileSizeString = (fileSizeInBytes: number) => {
+  // eslint-disable-next-line functional/no-let
+  let i = -1;
+  // eslint-disable-next-line functional/no-let
+  let size = fileSizeInBytes;
+  const byteUnits = [' kB', ' MB', ' GB', ' TB', 'PB', 'EB', 'ZB', 'YB'];
+  // eslint-disable-next-line functional/no-loop-statements
+  do {
+    size /= 1024;
+    i++;
+  } while (size > 1024);
+
+  return Math.max(size, 0.1).toFixed(1) + byteUnits[i];
+};
```

Comment:
`do - while` cycle are always to avoid i think is better to convert in this way kilobite to size
```suggestion
const getReadableFileSizeString = (fileSizeInBytes: number) => {
    if (kilobytes <= 0) {
        return "0 KB";
    }
    
    const units = ["KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"];
    const unitIndex = Math.floor(Math.log(kilobytes) / Math.log(1024));
    const size = kilobytes / Math.pow(1024, unitIndex);
    const formattedSize = size.toFixed(2);

    return `${formattedSize} ${units[unitIndex]}`;
};
```